packages/mdc-textfield/foundation.ts

```diff
@@ -345,115 +315,101 @@ class MDCTextFieldFoundation extends MDCFoundation {
 
   /**
    * Enables or disables the use of native validation. Use this for custom validation.
-   * @param {boolean} useNativeValidation Set this to false to ignore native input validation.
+   * @param useNativeValidation Set this to false to ignore native input validation.
    */
-  setUseNativeValidation(useNativeValidation) {
+  setUseNativeValidation(useNativeValidation: boolean): void {
     this.useNativeValidation_ = useNativeValidation;
   }
 
-  /**
-   * @return {boolean} True if the Text Field is disabled.
-   */
-  isDisabled() {
+  isDisabled(): boolean {
     return this.getNativeInput_().disabled;
   }
 
   /**
-   * @param {boolean} disabled Sets the text-field disabled or enabled.
+   * @param disabled Sets the text-field disabled or enabled.
    */
-  setDisabled(disabled) {
+  setDisabled(disabled: boolean): void {
```

Comment:
Don't need `: void` for foundation/component methods with no return

Reply:
Done